src/deephaven_ib/_tws/tws_client.py

```diff
@@ -182,6 +182,10 @@ def _build_table_writers() -> Dict[str, TableWriter]:
         table_writers["accounts_pnl"] = TableWriter(
             ["RequestId", "DailyPnl", "UnrealizedPnl", "RealizedPnl"],
             [dtypes.int64, dtypes.float64, dtypes.float64, dtypes.float64])
+        
+        table_writers["accounts_pnl_single"] = TableWriter(
```

Comment:
I think `_make_tables` also needs an entry for the new table.
https://github.com/deephaven-examples/deephaven-ib/blob/main/src/deephaven_ib/__init__.py#L570

This can be confirmed by looking at the output of `connection.tables` and `connection.tables_raw`.